packages/EasyAsync/src/Bridge/Laravel/Providers/EasyAsyncServiceProvider.php

```diff
@@ -44,7 +44,7 @@ public function register(): void
             DateTimeGeneratorInterface::class => DateTimeGenerator::class,
             JobFactoryInterface::class => JobFactory::class,
             JobLogFactoryInterface::class => JobLogFactory::class,
-            UuidGeneratorInterface::class => RamseyUuidGenerator::class,
+            UuidV4GeneratorInterface::class => RamseyUuidV4Generator::class,
```

Comment:
Hey @DKeeper, really good PR, just one thing. The idea with using easy-random here is to allow the application to have the control of which uuid generator to use. If you have a read of the easy-random documentation (https://packages.eonx.com/projects/eonx-com/easy-random/), you will see that the config to define which uuid v4 generator to use is handled there already.

In this package, the only thing we have to do is to require the `RandomGeneratorInterface` where we need it and call the `uuidV4()` method.

Of course you will have to explicitly define which UuidV4 generator to use in the test but that only for tests purposes.

Reply:
Done.